apps/frontend/src/pages/api/attestations.ts

```diff
@@ -0,0 +1,169 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import Axios from "axios";
+import web3 from "web3";
+
+type Query = {
+  query: string;
+  variables: string;
+};
+
+type QueryResponse = {
+  data: {
+    messageSents: [
+      {
+        message: string;
+      }
+    ];
+  };
+};
+
+const DOMAIN_ETH = 1;
+const DOMAIN_GOERLI = 5;
+const DOMAIN_AVAX = 43114;
+const DOMAIN_FUJI = 43113;
+const DOMAIN_ARB = 42161;
+const DOMAIN_ARBGOERLI = 421613;
+const DOMAIN_OP = 10;
+const DOMAIN_OPGOERLI = 420;
+
+const CCTP_API_ENDPOINT_TESTNET =
+  "https://iris-api-sandbox.circle.com/v1/attestations/";
+const CCTP_API_ENDPOINT_MAINNET =
+  "https://iris-api.circle.com/v1/attestations/";
+
+const CCTP_API_STATUS_COMPLETE = "complete";
+
+const axios = Axios.create();
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === "POST") {
+    const { data, sender } = req.body;
+    console.log("data:", data);
+    console.log("sender:", sender);
+
+    const destinationDomain = extractDestinationDomainFromMessage(data);
+    console.log("destinationDomain:", destinationDomain);
+    let graphQLEndpoint = "";
+    let cctpEndpoint = "";
+    switch (destinationDomain) {
+      case DOMAIN_ETH:
+        cctpEndpoint = CCTP_API_ENDPOINT_MAINNET;
+        graphQLEndpoint =
```

Comment:
Probably move this into dedicated config?

Reply:
done